web_embedding/cra-flutter/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "cra-flutter",
+  "version": "0.0.0",
+  "scripts": {
+    "prestart": "rimraf node_modules/.cache/",
+    "start": "react-scripts start",
+    "prebuild": "pushd flutter && flutter clean && flutter build web --output=../public/flutter && popd",
```

Comment:
> > Maybe it will be better to move web_embedding/ng-flutter/flutter directory to the web_embedding and share between different examples?

> This is a good idea, especially to unify the code we use in the Flutter side to communicate into JS (@JSExports, signaling with a custom event from the root of the app, etc...) however I'm concerned about having flutter outside of the Angular/ReactJS project directory; will the build system copy code from the outside? Will it make setup more weird than what it is right now?

If the `flutter` directory will be in the parent directory, this line will become 
```bash
pushd ../flutter && flutter clean && flutter build web --output=../cra-flutter/public/flutter && popd
```

And similar for the Angular app, we can have smth like this:

instead of 
```bash
pushd flutter && flutter clean && flutter build web && popd
```
we will have
```bash
pushd ../flutter && flutter clean && flutter build web --ouput=../ng-flutter/flutter && popd
```

These should be the only changes, besides documentation updates

Reply:
Yes, as long as we can put the build output of flutter within the reach of the "resources"/"assets" of the angular/react web apps, it might be interesting to share (but I think this is not super high priority, because it's rare that people looking for an example in a framework will look at others and say: hmmm, you should have de-duplicated this!, I think it's more like: OK, here's where my flutter app goes!)